### DISCUSSION TITLE
Documentation: "Ability to configure a custom OCPP server such as HA is being discontinued"

### DISCUSSION THREAD

**Discussion by Mark-Roly** [lbbrhzn/ocpp#577] [Q&A] [answered]
In the documentation, under Supported Devices, there is the statement:

_"(Ability to configure a custom OCPP server such as HA is being discontinued)"_

...is this referring to the model above the statement (EVNex) or is it referring to the models below it (Wallbox/Vestel/V2C)

Thanks

**Comment by drc38** [comment, marked as answer]
Evnex units only

**Comment by Mark-Roly** [comment]
Thank you - is there a source statement announcing this?

**Reply by drc38** [reply]
https://community.evnex.io/t/matching-charging-rate-to-solar-generation/37/4

**Reply by drc38** [reply]
Fyi there is an evnex cloud polling integration from @hardbyte now available https://github.com/hardbyte/ha-evnex
